w_common_tools/bin/compile_sass.dart

```diff
@@ -0,0 +1,4 @@
+import 'dart:async';
+import 'package:w_common_tools/src/bin/compile_sass.dart' as compiler;
+
+Future<Null> main(List<String> args) async => await compiler.main(args);
```

Comment:
If we don't need to be backward-compatible with something, `Future<void>` is preferred.